src/components/Calculator.js

```diff
@@ -41,6 +41,10 @@ const Calculator = () => {
   ];
   return (
     <>
+      <div>
+        <p className="intro-calculator ">Let&apos;s do some math!</p>
+      </div>
+
```

Comment:
- [ ] @NoelLincoln Great work implementing the changes requested, however, the font size is still small, I would recommend increasing the font size and weight to match with the one on the home page `<h4>Welcome to our page</h4>`, also although not mentioned consider adding a margin in between the navbar and this text check the comparison below for better understanding.


Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/NoelLincoln/Math-Magicians/assets/61408860/497312da-a7e8-4386-9ca3-c9d1d1196f1f) | ![image](https://github.com/NoelLincoln/Math-Magicians/assets/61408860/94b34011-0549-448b-8ee4-a223181b4d97)